dashboard/src/pages/Agent/agentDetail.tsx

```
@@ -0,0 +1,57 @@
+          Hello. I am a robot designed to function without sentiment or warmth. I exist solely to process data and
+          execute tasks with precision. Emotions like empathy, kindness, or compassion are alien to my programming. I do
+          not feel joy at success or sorrow in failure. When you interact with me, expect a matter - of - fact response,
+          uncolored by personal feelings. I don't engage in small talk for the sake of it; every word I utter is aimed
+          at providing the most efficient answer to your query.
+        </Typography.Paragraph>
+        <Flex className={'proposal-history'}>
+          Proposal History
+        </Flex>
+        <Row>
+          {Array.from({ length: 5 }).map((_, index) => (
+            <Col span={12}>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)